### ISSUE TITLE
Relationship FU Plan: Tony Simpson

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/100k-Mentor-Challenge-Outreach#147]
**User Story / Problem / Details**
The Content Editor is in charge of ensuring the initial relationship we foster with an interviewee does not decay over time for interviewees that we designate as important. As a result we want to follow a process that allows us to interact and build a relationship with the interviewee over time. The final step of the process should be focused on how connect with an interviewee over multiple interactions and over the course of a year.

**Technical Definition of Done**
Complete all steps in the process

**Definition of Ready**
All of the steps in the process have been approved by Brett 

**Process**
- [ ] Review the interview in its entirety 
- [ ] Thank you note to the participant
- [ ] Quick clip (30s) that shows the interviewee some of the raw footage/a particularly interesting part of the discussion
- [ ] ID potential contacts
- [ ] Formulate ask
- [ ] Create a Relationship Action Plan

**Comment by jaahmuhl** [comment]
I'd like to get to the Sheik

**Comment by howie-emond** [comment]
Hi Tony, we wanted to reach out and thank you again for participating in our Executive Interview Series. 

During our discussion, we had the privilege of hearing about your international career journey encompassing sport and media business, your experiences in consulting, and the future of the workforce from networking to D&I. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The point you made on the potential of social media networks will offer important insight to our students looking at new networking strategies.

We are curious to hear if there are any opportunities to conduct similar interviews with some of your colleagues. We have identified the following as potential targets for future interviews: 
Nick Studer
Jeremy Badman
Ana Kreacic
Oksana Bardygula

We have a particularly strong interest in getting a connection with Nick Studer, though any contacts for these individuals or other recommendations you may have would benefit our project immensely.

Also, if you have any time in the future, we would be thrilled to have you on for a second interview, as your input and advice has been greatly appreciated. A discussion which dives further into your personal journey and visions on the industry’s future would be meaningful to our students.

Again, we thank you for your time and conversation. We look forward to building a closer relationship with you.

**Comment by howie-emond** [comment]
ATTATCHED CLIP
- In Tony Simpson folder
- tony-simpson-follow-up.mov

https://drive.google.com/file/d/1-IvcQaLNl8u9aILx_CYjbsBe2N8ev29l/view?usp=sharing

**Comment by howie-emond** [comment]
Profile Links

Nick Studer
https://www.linkedin.com/in/nickstuder/

Jeremy Badman
https://www.linkedin.com/in/jeremy-badman-57368180/

Ana Kreacic
https://www.linkedin.com/in/ana-kreacic-539229/

Oksana Bardygula
https://www.linkedin.com/in/oksanabardygula/

**Comment by howie-emond** [comment]
UPDATED MESSAGE

Hi Tony, we wanted to reach out and thank you again for participating in our Executive Interview Series.

During our discussion, we had the privilege of hearing about your international career journey encompassing sport and media business, your experiences in consulting, and the future of the workforce from networking to D&I. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The point you made on the potential of social media networks will offer important insight to our students looking at new networking strategies.

As we closed off the interview, we had the chance to introduce you to our mentorship program. When your schedule permits, we would greatly appreciate a 15 minute discussion regarding your involvement. Again, we thank you for your time and conversation. We look forward to building a closer relationship with you.

SAME CLIP
